integrations/express/src/lib/static-rendering.ts

```diff
@@ -0,0 +1,84 @@
+import type { ApiReferenceOptions } from '@/types'
+import type { ReferenceConfiguration } from '@scalar/types'
+
+const DEFAULT_CONFIGURATION: Partial<ReferenceConfiguration> = {
+  _integration: 'express',
+}
+
+/**
+ * The HTML document to render the Scalar API reference.
+ */
+export function getHtmlDocument(
+  options: ApiReferenceOptions,
+  customTheme?: string,
+) {
+  return `
+    <!DOCTYPE html>
+    <html>
+      <head>
+        <title>Scalar API Reference</title>
+        <meta charset="utf-8" />
+        <meta
+          name="viewport"
+          content="width=device-width, initial-scale=1" />
+        <style>
+          ${options.theme ? null : (customTheme ?? '')}
+        </style>
+      </head>
+      <body>
+        ${getScriptTags(options)}
+      </body>
+    </html>
+  `
+}
+
+/**
+ * The script tags to load the @scalar/api-reference package from the CDN.
+ */
+export function getScriptTags(configuration: ApiReferenceOptions) {
+  return `
+      <script
+        id="api-reference"
+        type="application/json"
+        data-configuration="${getConfiguration(configuration)}">${getScriptTagContent(configuration)}</script>
```

Comment:
So we have more and more config options that cannot be passed through this way, should we instead push users towards passing config through JS instead of html?

Reply:
That said, it’s complex. :)) we’ll chat, got some thoughts

Reply:
we do have the esm api we use in a few integrations, maybe we can swindle that into the standalone. That would actually be nice because then we won't need to update the integrations in partner repos!